test/testnet/localnet/README.md

```diff
@@ -0,0 +1,94 @@
+# Remote Version of Localnet
+
+Remote version of the test network generated by running:
+
+```bash
+> make localnet-start
+```
+
+from the Tendermint repository. There are a few additional minor tweaks, however:
+
+1. The proxy app is preconfigured to `kvstore`
+2. Logging format is JSON
+3. `persistent_peers` is preconfigured
+4. `create_empty_blocks` is set to `false`
+5. `prometheus` is set to `true` to allow for collecting Prometheus metrics
+
+## Contents
+This folder contains the relevant prerequisites to deploy this test network to a
+set of predefined hosts. Here is a description of a subset of the layout of this
+folder's contents:
+
+```
+|_ Makefile           The primary entrypoint to interacting with the test network.
+|_ requirements.txt   Python requirements for running the deployment scripts.
+|_ config             Ansible configuration management.
+   |_ hosts           The inventory file for the hosts to which we will be deploying the network.
+   |_ (secrets.yml)   NOT PROVIDED. See the "Secrets" section below for details.
+|_ nodes              Tendermint config/data for each node in the `config/hosts` file.
+```
+
+## Prerequisites
+To be able to deploy this test network, you will need the following software
+installed:
+
+* make
+* Python 3
+* Go 1.11.4+
+* Tendermint source under `${GOPATH}/src/github.com/tendermint/tendermint`
+
+## Secrets
+In order to deploy this network, you will first need to set up a `secrets.yml`
+file inside the `config` folder with the following structure:
+
+```yaml
+secrets:
+  nodes:
+    node0:
+      address: "<redacted>"      # Validator address for this node
+      public_key: "<redacted>"
+      private_key: "<redacted>"
+      node_key: "<redacted>"     # Node private key (ed25519)
+      power: "1"
+    node1:
+      address: "<redacted>"
+      public_key: "<redacted>"
+      private_key: "<redacted>"
+      node_key: "<redacted>"
+      power: "1"
+    # etc., same for node2 and node3
+```
+
+There is a `secrets.yml.template` in the `config` folder for your convenience
+for you to fill out with your own values.
+
+## Deployment
+**NOTE**: This will **delete** any existing Tendermint configuration/data from
+the target nodes, so make sure you first back up your Tendermint configuration
+and data prior to running this command.
+
+Once you've backed up any important data, to deploy this test net, simply run:
+
+```bash
+# This assumes that you've got Tendermint in your ${GOPATH}, and it runs the
+# `make build-linux` command in the Tendermint source folder.
+> make build-tendermint
```

Comment:
If I recall, dependencies are not actually depepndent on `build-linux`, so for a fresh install, you need to run `make -C $GOPATH/src/github.com/tendermint/tendermint get_vendor_deps` first. Assuming you have `dep` installed and available.

Maybe it's quicker and painless to redirect the user to the "how to build the binary" documentation instead of recreating one here. It's the same principle.